packages/daemon/src/app/controller/job.ts

```diff
@@ -93,6 +123,49 @@ export class JobController extends BaseEventController {
     this.ctx.success();
   }
 
+  @get('/:id/parameters')
+  public async getParam(): Promise<void> {
+    const { id } = this.ctx.params;
+    const job = await this.pipelineService.getJobById(id);
+
+    if (job) {
+      this.ctx.success(job.params);
+    } else {
+      this.ctx.throw(HttpStatus.NOT_FOUND, 'no job found');
+    }
+  }
+
+  /**
+   * Run job with new param
+   */
+  @post('/:id/parameters')
+  public async updateParam(): Promise<void> {
```

Comment:
```suggestion
  public async updateParams(): Promise<void> {
```

Reply:
And I think we need to require parameters when running a job.

Reply:
I think it is possible to run a job without parameters. In this case, the job is run with the pipeline's parameters.

Reply:
Yea, what I mean we don't need an extra interface to update the parameters, it should be an optional argument on run.

Reply:
Merged them into one